doc/sphinx/zrefs.bib

```diff
@@ -556,39 +394,7 @@ @ARTICLE{magatti01a
   pages = {4011-4021},
```

Comment:
May be beyond the scope of this PR, but actually ALL page-ranges should use n-dashes (e.g., here, `pages = {4011--4021}`).

Reply:
Done.